lib/util/enum_util.ex

```diff
@@ -8,7 +8,10 @@ defmodule Antikythera.EnumUtil do
   """
 
   @type context :: any
-  @type item :: any
+  @type item    :: any
+  @type element :: any
```

Comment:
Although originally `item` is used to represent each element in an `Enum.t`, `element` looks better.
So let's remove `@type item` here and use `element` throughout this module.
(Actually it's slightly backward-incompatible change to remove a public type, but it's OK to just remove this, since it's almost not used.)

Also this type should have been an alias to `Enum.element`.
```ex
@type element :: Enum.element
```

Reply:
fixed in 57e0df7